modules/example-apache-httpd-configuration.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * authentication/identity_providers/configuring-ldap-identity-provider.adoc
+
+[id="example-apache-httpd-configuration_{context}"]
+= Example Apache HTTPD configuration for basic identity providers
+
+The basic identify provider (IDP) configuration in {product-title} 4 requires
+that the IDP server respond with JSON for success and failures. You can use CGI
+scripting in Apache HTTPD to accomplish this. This section provides examples.
+
+== File requirements
+
+These are the requirements for the files you create on an Apache HTTPD web
+server:
+
+* `login.cgi` and `fail.cgi` must be executable (`chmod +x`).
+* `login.cgi` and `fail.cgi` must have proper SELinux contexts if SELinux is
+enabled: `restorecon -RFv /var/www/cgi-bin`, or ensure that the context is
+`httpd_sys_script_exec_t` using `ls -laZ`.
+* `login.cgi` is only executed if your user successfully logs in per `Require
+and Auth` directives.
+* `fail.cgi` is executed if the user fails to log in, resulting in an `HTTP 401`
+response.
+
+== `/etc/httpd/conf.d/login.conf`
```

Comment:
I think using different header types for these files names would make sense. Right now they jump out too much on the page.

Reply:
Yeah,I agree that,I dont think the file location should be prefixed by `==`,can you remove it?

Reply:
@bostrt @barleyer Updated. Thank you!